app/views/home/_topannounce.html.erb

```diff
@@ -0,0 +1,26 @@
+<% if @mysite == "autolab.cs.cmu.edu" %>
```

Comment:
Do we maybe want to have this file be `app/view/home/_topannounce.html.erb.template` and have this file be gitignored? Otherwise, people will get merge conflicts if they try to update this file and we release an update.